app/assets/javascripts/views/searches/results/results_default.js

```diff
@@ -53,18 +59,31 @@
       };
     }
 
+    function includeLingIds(columns){
+      if (columns["ling_0"]) {
+        columns["ling_0_id"] = "1";
+      }
+      if (columns["ling_1"]) {
+        columns["ling_1_id"] = "1";
+      }
+    }
+
     function defaultMapping(columns, entry){
       var func_dict = {
+        'ling_0_id' : getLingID('parent'),
         'ling_0'    : getLing('parent'),
         'property_0': getProperty('parent'),
         'value_0'   : getValue('parent'),
         'example_0' : getExamples('parent'),
+        'ling_1_id' : getLingID('child'),
         'ling_1'    : getLing('child'),
         'property_1': getProperty('child'),
         'value_1'   : getValue('child'),
         'example_1' : getExamples('child')
       };
 
+      includeLingIds(columns);
```

Comment:
This is the point that I include ling ids to column result. After that it possible to extract the lings ids for the feature tests